src/levanter/compat/hf_checkpoints.py

```diff
@@ -757,6 +759,9 @@ def save_pretrained(
                 upload_to_hf = self.reference_checkpoint
             if not isinstance(upload_to_hf, bool):
                 assert isinstance(upload_to_hf, (str, RepoRef))
+                if isinstance(upload_to_hf, str) and not repo_exists(upload_to_hf, repo_type="model"):
```

Comment:
I assume a `RepoRef` can only exist when an actual repo also exists in HF.

Reply:
makes sense to me!